Automattic/wp-calypso
# Promote Post report data tab

#### Proposed Changes
- Added Stats Page to Promoted Posts Page (currently disabled)

#### Testing Instructions
- Open https://wordpress.com/advertising
- Click on Campaigns button. Currently it shouldn't display values as it needs to be hydrated by future development

It should be similar to this:
![image](https://user-images.githubusercontent.com/43957544/186546161-aede4a6c-e491-4bf1-bab0-2721d20feec8.png)

#### Pre-merge Checklist

Related to #
